Open Judge System/Data/OJS.Data/Migrations/DefaultMigrationConfiguration.cs

```diff
@@ -357,7 +357,7 @@ protected void SeedSubmissionTypes(OjsDbContext context)
                 {
                     Name = "C++ Zip File",
                     CompilerType = CompilerType.CPlusPlusZip,
-                    AdditionalCompilerArguments = "-pipe -mtune=generic -O3 -static-libgcc -static-libstdc++",
```

Comment:
Did you test with lower optimization level? What problems have you encountered with this optimization level and the lower ones?